compiler-rt/test/builtins/Unit/floattitf_test.c

```diff
@@ -7,7 +7,7 @@
 #include <float.h>
 #include <stdio.h>
 
-#if defined(CRT_HAS_128BIT) && defined(CRT_LDBL_128BIT)
+#if defined(CRT_HAS_TF_MODE)
```

Comment:
This one actually fails without and IEEE float128. Right now that doesn't matter but it will start failing once CRT_HAS_TF_MODE gets set with non-IEEE TF floats (powerpc64).

Reply:
Correct guard added in  https://github.com/llvm/llvm-project/pull/68132.